website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md

```diff
@@ -6,4 +6,628 @@ description: >
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.interactions.PointerInput;
+
+public class scrollToElement {
+
+  public static void main(String[] args) {
+    WebDriver driver = new ChromeDriver();
+    try {
+      driver.get("https://crossbrowsertesting.github.io/selenium_example_page.html");
+      WebElement element = driver.findElement(By.id("closepopup"));
+
+      Actions actions = new Actions(driver);
+      actions.scroll(0,
+                     0,
+                     0,
+                     0,
+                     PointerInput.Origin.fromElement(element))
+        .perform();
+    } finally {
+      driver.quit();
+    }
+  }
+}
+  {{< /tab >}}
+  {{< tab header="Python" >}}
+from selenium import webdriver
+from selenium.webdriver import ActionChains
+from selenium.webdriver.common.by import By
+
+driver = webdriver.Chrome()
+driver.get("https://crossbrowsertesting.github.io/selenium_example_page.html")
+element = driver.find_element(By.ID, "closepopup")
+
+ActionChains(driver).scroll(0, 0, 0, 0, origin=element).perform()
+
+driver.quit()
+  {{< /tab >}}
+  {{< tab header="CSharp" >}}
+  {{< /tab >}}
+  {{< tab header="Ruby" >}}
+require 'selenium-webdriver'
+
+begin
+  driver = Selenium::WebDriver.for :chrome
+  driver.get 'https://crossbrowsertesting.github.io/selenium_example_page.html'
+  element = driver.find_element(:id, 'closepopup')
+  driver.action.scroll_to(element).perform
+ensure
+  driver.quit
+end
+  {{< /tab >}}
+  {{< tab header="JavaScript" >}}
+const { Builder, By } = require('selenium-webdriver');
+
+(async function scrollToElement() {
+  let driver = await new Builder().forBrowser('chrome').build();
+
+  try {
+    await driver.get('https://crossbrowsertesting.github.io/selenium_example_page.html');
+    element = await driver.findElement(By.id('closepopup'));
+    await driver.actions().scroll(0, 0, 0, 0, element).perform();
+  }
+  finally {
+    await driver.quit();
+  }
+})();
+
+  {{< /tab >}}
+  {{< tab header="Kotlin" >}}
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.chrome.ChromeDriver
+import org.openqa.selenium.interactions.Actions
+import org.openqa.selenium.interactions.PointerInput
+
+fun main() {
+    val driver: WebDriver = ChromeDriver()
+    try {
+        driver["https://crossbrowsertesting.github.io/selenium_example_page.html"]
+        val element = driver.findElement(By.id("closepopup"))
+        val actions = Actions(driver)
+        actions.scroll(
+            0,
+            0,
+            0,
+            0,
+            PointerInput.Origin.fromElement(element)
+        ).perform()
+    } finally {
+        driver.quit()
+    }
+}
+  {{< /tab >}}
+{{< /tabpane >}}
+
+## Scroll by given amount from element 
+
+Scrolls to the element by scrolling the viewport. This way the element is at the bottom. Scrolls the viewport further by the given amount i.e. horizontal and vertical offsets.
```

Comment:
Hi @pujagani,

can we limit 100 chars per line?

Reply:
Sure